webui/react/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,193 @@
+import { Breadcrumb, Table } from 'antd';
+import React, { useCallback, useEffect, useState } from 'react';
+
+import ExperimentIcons from 'components/ExperimentIcons';
+import Grid, { GridMode } from 'components/Grid';
+import Link from 'components/Link';
+import Page from 'components/Page';
+import ProjectCard from 'components/ProjectCard';
+import Section from 'components/Section';
+import { experimentNameRenderer, relativeTimeRenderer } from 'components/Table/Table';
+import { paths } from 'routes/utils';
+import {
+  getCommands,
+  getExperiments,
+  getJupyterLabs,
+  getProjectsByUserActivity,
+  getShells,
+  getTensorBoards,
+} from 'services/api';
+import Icon from 'shared/components/Icon/Icon';
+import { dateTimeStringSorter } from 'shared/utils/sort';
+import { useAuth } from 'stores/auth';
+import { ShirtSize } from 'themes';
+import { CommandTask, DetailedUser, ExperimentItem, Project } from 'types';
+import { Loadable } from 'utils/loadable';
+
+const Dashboard: React.FC = () => {
+  const [experiments, setExperiments] = useState<ExperimentItem[]>([]);
+  const [tasks, setTasks] = useState<CommandTask[]>([]);
+  const [projects, setProjects] = useState<Project[]>([]);
+  const [submissions, setSubmissions] = useState<Submission[]>([]);
+  const [canceler] = useState(new AbortController());
+  const loadableAuth = useAuth();
+  const authUser = Loadable.match(loadableAuth.auth, {
+    Loaded: (auth) => auth.user,
+    NotLoaded: () => undefined,
+  });
+
+  type Submission = ExperimentItem & CommandTask;
+
+  const fetchTasks = useCallback(
+    async (user: DetailedUser) => {
+      const [commands, jupyterLabs, shells, tensorboards] = await Promise.all([
+        getCommands({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getJupyterLabs({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getShells({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getTensorBoards({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+      ]);
+      const newTasks = [...commands, ...jupyterLabs, ...shells, ...tensorboards];
+      setTasks(newTasks);
+    },
+    [canceler],
+  );
+
+  const fetchExperiments = useCallback(async (user: DetailedUser) => {
+    const response = await getExperiments({
+      orderBy: 'ORDER_BY_DESC',
+      sortBy: 'SORT_BY_START_TIME',
+      userIds: [user.id],
+    });
+    setExperiments(response.experiments);
```

Comment:
I think we should have some kind of limit in this call to avoid fetching all experiments